frontend/src/components/MealSection.js

```diff
@@ -1,21 +1,30 @@
 import MealItem from './MealItem'
 import dropdownIcon from '../assets/icons/dropdown.svg'
+import dropupIcon from '../assets/icons/dropup.svg'
+import React, { useState } from 'react';
 
 const MealSection = (props) => {
+    const [isOpen, setIsOpen] = useState(true);
+    const [items, setItems] = useState(props.items);
```

Comment:
State isn't necessary here for items because we never update it. 

On line 21 instead of saying `items.map(...)` you can just use `props.items.map(...)`